docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle Database connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Oracle Database details](#gather-oracle-database-details)
+2. [Configure Oracle Database connector](#configure-oracle-database-connector)
+
+#### Gather Oracle Database details
+
+Collect the information that is required to connect to your Oracle Database:
+
+- The server host address where the Oracle Database is hosted.
+- The port where the Oracle Database is hosted.
+- The username the connector will use to log in to the Oracle Database.
+- The password the connector will use to log in to the Oracle Database.
+- The database name where the connector will query data.
```

Comment:
Do we only support database name, or also service name?

Reply:
No, currently we are not supporting the service name. But we are using SID to connect with the oracle database. for simplicity change database to SID.